components/layouts/dashboard/dashboardDrawer/dashboardDrawerItem/DashboardDrawerItem.tsx

```diff
@@ -0,0 +1,67 @@
+import styles from './DashboardDrawerItem.module.css';
+import {
+    Collapse,
+    ListItemButton, ListItemIcon, ListItemText, Tooltip,
+} from "@mui/material";
+import React from "react";
+import {ExpandLess, ExpandMore} from "@mui/icons-material";
+import {PopupState} from "material-ui-popup-state/core";
+import {bindToggle} from "material-ui-popup-state/hooks";
+
+
+export interface IDashboardDrawerItem {
+    text: string;
+    icon: React.ReactElement;
+    drawerOpen: PopupState;
+    dropDownType: string;
+
```

Comment:
drop down type should instead be a combined literal type like 
```suggestion
    dropDownType: "child" | "parent";

```